Source/Core/DolphinQt/Debugger/MemoryViewWidget.cpp

```diff
@@ -122,6 +126,27 @@ class MemoryViewTable final : public QTableWidget
     }
   }
 
+  void OnItemChanged(QTableWidgetItem* item)
+  {
+    QString text = item->text();
+    MemoryViewWidget::Type type =
+        static_cast<MemoryViewWidget::Type>(item->data(USER_ROLE_VALUE_TYPE).toInt());
+    std::vector<u8> bytes = m_view->ConvertTextToBytes(type, text);
+
+    u32 address = item->data(USER_ROLE_CELL_ADDRESS).toUInt();
+    u32 end_address = address + static_cast<u32>(bytes.size()) - 1;
+    AddressSpace::Accessors* accessors = AddressSpace::GetAccessors(m_view->GetAddressSpace());
+
+    if (accessors->IsValidAddress(address) && accessors->IsValidAddress(end_address) &&
+        !bytes.empty())
```

Comment:
Please reorder these so that the cheap check (the `bytes.empty()`) is performed first. In fact, you might want to check `if (bytes.empty())` and early return right after the `ConvertTextToBytes()` call.

Reply:
I don't 100% understand the accessor stuff.  Does checking for a valid address work for all address spaces?

Reply:
Oh wait. I did this because we need to send an update no matter what, So it should be at the beginning.